templates/home.tmpl

```diff
@@ -1,6 +1,6 @@
 {{template "base/head" .}}
 <div role="main" aria-label="{{if .IsSigned}}{{.locale.Tr "dashboard"}}{{else}}{{.locale.Tr "home"}}{{end}}" class="page-content home">
-	<div class="ui stackable middle very relaxed page grid">
+	<div class="ui middle very relaxed page gt-mb-5">
```

Comment:
It's not a `grad` anymore, does the code `sixteen wide column`  below make sense? Or should they be cleaned up?

Reply:
No, I think those classes are dead now. But still, I left them in because this seems like a common target for customization, and I don't want to unnecessarily break those.

Reply:
Since you have removed the `grid` on its parent, keeping the dead code doesn't help about avoiding "breaking", instead it might really cause breaks if the customization depends on the `grid`.

And, I worry about that the dead code might cause other style conflicts in the future or unnecessary technical debt / copy-paste bug in the future.

So I guess we should remove dead code as soon as possible.

Reply:
I guess we could also just add `gt-px-0` to the classes and keep the old ones. Same effect and even more compatible.

Reply:
TBH I do not think so ....

If a developer reads code:

```html
<div class="ui ...."> 
   <div class="six wide column">
   </div>
</div>
```

They will think about: OK , the inner div is designed to take 6/12 or 6/24 of the screen.

But the truth is the `"six wide column"` is a no-op now. It really surprises developers, I do not like such surprise.

There are already a lot of surprises in Gitea's codebase. To improve maintainability and readability, let's try to avoid surprises?

Reply:
Maybe you could file a new PR with your envisioned changes. I don't quite understand what is requested anymore.

Reply:
No problem.  Improve home page template #23856 


Major changes:

1. The old `<div class="ui"><div class="six wide column ..."> </div></div>` doesn't have affect any more
    * So clean them, and remove other unnecessary elements/styles.
2. Add padding for narrow view.